### DISCUSSION TITLE
How can I write test code in a custom build configuration without `Debug` postfix?

### DISCUSSION THREAD

**Discussion by havilog** [pointfreeco/swift-composable-architecture#1990] [Q&A] [answered]
I tried to write test code with composable architecture embedded by spm.

I'm using custom build configuration named `SomethingSandbox`.

but I've found that `#if DEBUG` doesn't work when build scheme doesn't have `Debug` postfix like `SomethingDebug`

how can I solve this problem?

**Comment by lukeredpath** [comment, marked as answer]
You can’t really, SPM only really knows about two configurations - Debug and Release and so you need to suffix it to get it to recognise it.
